### DISCUSSION TITLE
When did agent computer burn down?

### DISCUSSION THREAD

**Discussion by NubeBuster** [Ylianst/MeshCentral#6380] [General]
Hi, one of my mesh agent computers was in a building that burnt down. I'd like to see if I can pinpoint when exactly the computer lost connection or burned down.

I exported the data from the stats page but it's not super high resolution on the 30 day timeframe. Is there some way I can get the data in a higher resolution even though it was 2 weeks ago? I've looked in the .db files but they are quite obscure.

Thanks in advance!

**Comment by NubeBuster** [comment]
There is a Last seen display on the website. Found it!

**Reply by si458** [reply]
sorry to hear about your building!
yes the should be a `Last Seen` on the General tab of the device in the web ui
and this is generally when the device was last connected to meshcentral 
and either the internet to that device dropped OR computer shutdown (or sadly in your case, caught fire 🔥 )

also in the `Details` tab the should be last agent connection and last boot up time
these might be helpful for you too!

**Reply by NubeBuster** [reply]
I have the computers configured to reboot at 4:30. The last boot time was 4:31 the day before, as usual. But the last connection was also 4:32 on the day of the fire. This makes it seem like the computer lost connection right after booting up. I do have to assume this is a coincidence though. It was not my building, but according to news sources the fire was discovered either at 4:00 or 4:15 so it kinda checks out.

**Reply by si458** [reply]
it might of been that meshagent was in the middle of running all its scripts etc tochange the last boot time, etc when it shut off!
as the last connection was 2 mins after a normal reboot,
but wasnt enough time for it to update the last boot up time!
sorry again tho about the fire!
